tutorials/physics/large_world_coordinates.rst

```diff
@@ -229,6 +229,23 @@ some limitations when it comes to 3D rendering precision:
 - :ref:`Triplanar mapping <doc_standard_material_3d_triplanar_mapping>` doesn't
   benefit from increased precision. Materials using triplanar mapping will exhibit
   visible jittering when far away from the world origin.
+- World space coordinates in a shader ``fragment()`` function can't be
+  reconstructed from view space, for example:
```

Comment:
This should have a qualifier that it only applies to double precision builds

Reply:
Isn't that implied by being on the Large world coordinates page? The whole page is about double precision builds, which are also called "large world coordinates".

Reply:
Yes, please ignore me. I didn't look at the title of the page

Reply:
I specified it anyway. It doesn't hurt and there's a chance that the section is referenced out of context.